roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2

```diff
@@ -0,0 +1,181 @@
+          "format": "password",
+          "info": "The secret of this Resource Server for authentication purposes."
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "fetch": "scopes"
+          },
+          "info": "The allowed scopes for this Resource Server."
+        }
+      },
+      "patternProperties": {
+        "^name:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "Set the name of the service. Format: 'Service Name | Supplier'"
+        },
+        "^description:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The description of the service."
+        },
+        "^OrganizationName:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The formal name of the organization. e.g. Service by University of Harderwijk."
+        },
+        "^OrganizationDisplayName:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The friendly name of the organization. e.g. University of Harderwijk."
+        },
+        "^NameIDFormats:([0-2]{1})$": {
+          "$ref": "#/definitions/NameIDFormat",
+          "multiplicity": 3,
+          "info": "The NameIDFormat(s) this service provider supports. This can be transient, persistent or unspecified."
+        },
+        "^contacts:([0-3]{1}):surName$": {
+          "type": "string",
+          "multiplicity": 4,
+          "sibblingIndependent": true,
+          "info": "The surname of the contact. e.g. for 'John Doe', 'Doe' is the surname."
+        },
+        "^contacts:([0-3]{1}):givenName$": {
+          "type": "string",
+          "multiplicity": 4,
+          "sibblingIndependent": true,
+          "info": "The given name of the contact. e.g. for 'John Doe', 'John' is the given name."
+        },
+        "^contacts:([0-3]{1}):emailAddress$": {
+          "type": "string",
+          "format": "local-email",
+          "multiplicity": 4,
+          "sibblingIndependent": true,
+          "info": "The e-mail address; syntax in accordance with RFC 5322. e.g. m.l.vermeegen@university.example.org."
+        },
+        "^contacts:([0-3]{1}):telephoneNumber$": {
+          "type": "string",
+          "multiplicity": 4,
+          "sibblingIndependent": true,
+          "info": "The telephone number of the contact."
+        },
+        "^contacts:([0-3]{1}):contactType$": {
+          "type": "string",
+          "enum": [
+            "technical",
+            "support",
+            "administrative",
+            "billing",
+            "other"
+          ],
+          "multiplicity": 4,
+          "sibblingIndependent": true,
+          "info": "From the pull down, select the type of contact."
+        }
+      },
+      "required": [
+        "name:en",
+        "secret",
+        "scopes"
```

Comment:
Is this really required?

Reply:
Good question. We need 1 name and without secret the RS can not introspect. Scopes we can do without I guess.